repository-dispatch/index.ts

```diff
@@ -6,13 +6,24 @@ import { getInput } from '@actions/core'
 class RepositoryDispatch extends Action {
 	id = 'RepositoryDispatch'
 
+	constructor() {
+		const token = getInput('token')
+		if (!token && getInput('event_type') === 'oss-pull-request') {
```

Comment:
I am not really sure that the token would be empty for forks or just have `read` permissions.
https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token suggest read

Reply:
@AgnesToulet alright. I just found this one and thought it might be the one used https://github.com/grafana/grafana-enterprise/blob/main/.github/workflows/enterprise-pr-check.yml#L20